git/branch_exercise.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Branching & Merging Exercise</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../common/css/bootstrap.min.css">
+</head>
+<body>
+    <div class="container">
+        <h1>Exercise: Branches!</h1>
+        <p>In this exercise, you will adding new features to your pet adoption website.  You will be adding a gallery page, and if there is time, a contact page.</p>
+
+        <p>As you read through these steps, think about when would be a good time to commit.  As you execute the steps, YOU get to decide when to commit.</p>
+
+        <h4>Set up</h4>
+        <ol>
+            <li>Switch back to your project folder with your pet adoption website.</li>
+            <li>Use git status to double check to make sure you have no modified/uncommitted changes</li>
```

Comment:
IMHO.
1. check branch
2. check status
Otherwise you may end up looking at a bunch of status changes which aren't for the right branch.

Wording on checking the git status feels odd. Suggested change:
`Use git status and confirm you have no modified/uncommitted changes`